src/controllers/BasicController.js

```diff
@@ -20,7 +20,7 @@ var BasicController = function(params) {
 };
 
 BasicController.prototype.doc = function() {
-  return (this.frameSelector) ? document.querySelector(this.frameSelector).contentWindow.document : document;
+  return (this.frameSelector && document.querySelector(this.frameSelector).tagName === "IFRAME") ? document.querySelector(this.frameSelector).contentWindow.document : document;
```

Comment:
I feel like we should throw an error instead of just returning the document.  Is there a case where frameSelector would be non-null and we'd still want to return the regular doc?